### PR TITLE
Fixed issue #16665: Question index not shown if survey mode is set to "inherit"

### PR DESCRIPTION
**Pull request by gabrieljenik**
Reviewed code

### CODE REVIEW

File: application/helpers/questionIndexHelper.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ private function __construct($iSurveyId)
         $oSurvey = \Survey::model()->findByPk($this->iSurveyId);
         if ($oSurvey) {
             $this->indexType = $oSurvey->questionindex;
```
Comment (Shnoulle) [inline application/helpers/questionIndexHelper.php:68]: `$oSurvey->aOptions['questionindex'];` ?

Comment (gabrieljenik) [reply application/helpers/questionIndexHelper.php]: Updated

Comment (gabrieljenik) [reply application/helpers/questionIndexHelper.php]: Updated!

Comment (Shnoulle) [reply application/helpers/questionIndexHelper.php]: PS : i didn't really check, just see it can be ;)

**Comment by maziminke** [comment]
@cdorin93 Can we please include this fix at the next 4.x release? A customers needs it.

**Review by eddylackmann — APPROVED** [review]

**Pull request merged**